src/js/components/AppListItemComponent.jsx

```diff
@@ -227,7 +303,7 @@ var AppListItemComponent = React.createClass({
     return (
       // Set `title` on cells that potentially overflow so hovering on the
```

Comment:
Marginal note: We should remove these comments as the respective code was moved.